### ISSUE TITLE
Accessibility swarm on the footer

### ISSUE THREAD

**Issue by ghost** [bbc/simorgh#1905]
**Is your feature request related to a problem? Please describe.**
Carry out a two devs accessibility swarm on the footer component. Check out multi language links are read out correctly by screen readers.

**Describe the solution you'd like**
- Check acceptance criteria on WSFRONTPGS-24
- Go through the [accessibility checklist](https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers) and confirm everything is working as expected and fix any issues that might crop up.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

**Comment by dr3** [comment]
❓ = Not done
✅ = Done, pass
❌ = Done, fail

❓ ZoomText Magnifier/Reader Latest Version
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Screen Magnifier with Screen Reader capabilities)

❓ Dragon Naturallyspeaking Version 13
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Speech Recognition)

❓ JAWS Version 17
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Screen Reader)

❓ Read&Write Latest Version
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Reading Solution)

✅ VoiceOver iOS (iPhone/iPad) Latest Version
With Safari (Latest Version) on iOS (Latest Version)
(Screen Reader)

❓ NVDA (Latest Version)
With Firefox (Latest Version) on Windows (XP/Vista/7/8/10)
(Screen Reader)

**Comment by j-pendlebury** [comment]
❓ = Not done
✅ = Done, pass
❌ = Done, fail
⭐️ = Questions

✅ ZoomText Magnifier/Reader Latest Version
With Internet Explorer 11 on Windows (~XP/Vista/~ 7 ~/8/10~)
(Screen Magnifier with Screen Reader capabilities)

✅ Dragon Naturallyspeaking Version 13
With Internet Explorer 11 on Windows (~XP/Vista/~ 7 ~/8/10~)
(Speech Recognition)

⭐️ JAWS Version 17
With Internet Explorer 11 on Windows (~XP/Vista/~ 7 ~/8/10~)
(Screen Reader)
➡️ After the last link on Igbo, JAWS says "BLANK". Is that normal?
➡️ Before branding, JAWS says something along the lines of "Content information". What is that?

✅ Read&Write Latest Version
With Internet Explorer 11 on Windows (~XP/Vista/~ 7 ~/8/10~)
(Reading Solution)

✅ VoiceOver iOS (iPhone/iPad) Latest Version
With Safari (Latest Version) on iOS (Latest Version)
(Screen Reader)

✅ NVDA (Latest Version)
With Firefox (Latest Version) on Windows (~XP/Vista/~ 7 ~/8/10~)
(Screen Reader)

✅ TalkBack (Latest Version)
With Chrome (Latest Version) on Android (Latest Version)
(Screen Reader)

**Comment by j-pendlebury** [comment]
Dragon to be completed tomorrow by @david-boydell ⭐️

**Comment by j-pendlebury** [comment]
❌ Issues with manual checks:

- Dark high contrast mode, the contrast between links is not the best
![image](https://user-images.githubusercontent.com/13763899/60676148-0e044080-9e76-11e9-84b6-849ae12eea21.png)
![image](https://user-images.githubusercontent.com/13763899/60676159-18bed580-9e76-11e9-9986-019c2b07bc08.png)

- Resizing the font to 72 makes the branding look very small
![image](https://user-images.githubusercontent.com/13763899/60676190-2ecc9600-9e76-11e9-8534-cfef51c7b5c6.png)

**Comment by sareh** [comment]
@jblaut The "Content information" is due to the role attribute `role="contentinfo"` that is the recommended landmark for the footer.
Jaws says "blank" whenever there's a blank line or space between content. It's to reflect the visual space between content in audible form.

**Comment by j-pendlebury** [comment]
Thanks @sareh 🍪⭐️

**Comment by dr3** [comment]
Brand size covered by https://github.com/bbc/psammead/issues/784

**Comment by greenc05** [comment]
@jblaut "Dark high contrast mode, the contrast between links is not the best" - That's just due to the colours you have selected?
